cmd/create.go

```diff
@@ -18,9 +18,11 @@ package cmd
 
 import (
 	"fmt"
+	"os"
+
 	pkgcmd "github.com/kedgeproject/kedge/pkg/cmd"
```

Comment:
can you add an empty line between kedge packages and 3rd party packages?
It would be good to start organizing imports in 3 blocks (stdlib, kedge pkgs, 3rd party)